client/src/components/menubar/index.js

```diff
@@ -419,6 +432,21 @@ class MenuBar extends React.Component {
             />
           </Tooltip>
         </div>
+        <Tooltip
+          content="View the centroids for all values"
+          position="bottom"
+          disabled={graphInteractionMode === "zoom"}
+        >
+          <Button
```

Comment:
You might consider disabling this button when there is no active color-by.  It is confusing that when you click on it, in the default (not colored by) mode, nothing happens.

Reply:
I'm a little torn between the lack of results when clicking on the centroid toggle and the possible confusion when having the side-effect of disabling centroids when removing active color-by. Any thoughts @liaprins-czi ?

Reply:
Fixed using Lia's suggestion